tests/app/models/BaseModelTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use DB\SQL;
+
+abstract class BaseModelTest extends TestCase
+{
+
```

Comment:
Please remove the extra new line here.

Over all this class looks much sleeker now!  Good work!